engines/gob/console.cpp

```diff
@@ -192,4 +216,17 @@ bool GobConsole::cmd_listArchives(int argc, const char **argv) {
 	return true;
 }
 
+bool GobConsole::cmd_EasterEgg(int argc, const char **argv) {
+	debugPrintf("Adibou 2\n");
+	debugPrintf("-----------\n");
+	debugPrintf("When creating a character with a name from the Coktel team (e.g. 'Cathy'), the birth date is automatically filled with the real world value.\n");
```

Comment:
It would be good to sort these alphabetically (i.e. move the Adibou 1 entries before the Adibou 2 ones)